src/main/java/dev/openfeature/sdk/Client.java

```diff
@@ -10,7 +10,7 @@ public interface Client extends Features {
 
     /**
      * Return an optional client-level evaluation context.
-     * @return {@link EvaluationContext}
+     * @return {@link MutableContext}
      */
     EvaluationContext getEvaluationContext();
```

Comment:
These docs say it should return a MutableContext? I'm thinking the docs are wrong?

Reply:
Ya, bulk replace. I will fix.

Reply:
I think this is the same as https://github.com/open-feature/java-sdk/pull/112#discussion_r988374129